fhir-install/src/main/docker/ibm-fhir-server/bootstrap.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# ----------------------------------------------------------------------------
+# (C) Copyright IBM Corp. 2021
+#
+# SPDX-License-Identifier: Apache-2.0
+# ----------------------------------------------------------------------------
+
+##############################################################################
+# Description:
+# This script is the entrypoint used by the ibm-fhir-server docker image, and
+# optionally bootstraps a derby database prior to running the IBM FHIR Server.
+
+set -e -o pipefail
+
+##############################################################################
+# The global variables used are:
+
+SCRIPT_NAME="$(basename ${BASH_SOURCE[0]})"
+
+CUR_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+FHIR_PERSISTENCE_SCHEMA_CLI_LOCATION="/opt/ibm-fhir-server/tools"
+
+PERFORM_BOOTSTRAP_DB=${BOOTSTRAP_DB}
+[ -z "${BOOTSTRAP_DB}" ] && PERFORM_BOOTSTRAP_DB="false"
+
+##############################################################################
+# Helper Functions
+
+# info - - local function to echo info message
+# ARGUMENTS:
+#   String of message
+function info {
+    echo "${SCRIPT_NAME} - [INFO]: $(date +"%Y-%m-%d_%T") - ${1}"
+}
+
+# _call_derby_db - local function to call derby database
+# ARGUMENTS:
+#   String of additional parameters
+function _call_derby_db {
+    /opt/java/openjdk/bin/java -jar ${FHIR_PERSISTENCE_SCHEMA_CLI_LOCATION}/fhir-persistence-schema-*-cli.jar \
+        --prop "db.create=Y" \
+        --prop "db.database=derby/fhirDB" \
+        --db-type derby \
+        ${1} 2>&1
```

Comment:
Why would we take `${1}` what are the. additional parameters we would want to take?

Reply:
initially it was used so we could call `create-schemas` and then `update-schema`, but then I had troy remove the `create-schemas` step since it wasn't needed for embedded derby
personally, I think its fine to keep the helper this way in case we want to adapt the script to bootstrap other dbs in the future